app/controllers/devise_token_auth/passwords_controller.rb

```diff
@@ -181,5 +173,16 @@ def password_resource_params
     def render_not_found_error
       render_error(404, I18n.t('devise_token_auth.passwords.user_not_found', email: @email))
     end
+
+    def validate_redirect_url_param
+      # give redirect value from params priority
+      @redirect_url = params.fetch(
+        :redirect_url,
+        DeviseTokenAuth.default_password_reset_url
+      )
+
+      render_create_error_missing_redirect_url unless @redirect_url
```

Comment:
It seems I wrote about it and didn't add comment.
```suggestion
      return render_create_error_missing_redirect_url unless @redirect_url
```